packages/adapters-library/src/core/adapters/UniswapV2PoolForkAdapter.ts

```diff
@@ -680,8 +674,6 @@ export abstract class UniswapV2PoolForkAdapter implements IProtocolAdapter {
               { ...token0, address: getAddress(token0.address) },
               { ...token1, address: getAddress(token1.address) },
             ],
-            token0: getAddress(token0.address),
-            token1: getAddress(token1.address),
```

Comment:
Not part of the metadata. This was being leaked.

Reply:
Wrong. This needs to be reverted before merging.

SQL queries do not guarantee the order, even though in practical terms they will be in order. We should not rely on that.